src/main/java/org/cactoos/io/Joined.java

```diff
@@ -40,15 +40,25 @@
      */
     private final Iterable<Input> inputs;
 
+    /**
+     * Ctor.
+     * @param inputs Iterable of inputs
+     */
+    public Joined(final Iterable<Input> inputs) {
+        this.inputs = inputs;
+    }
+
     /**
      * Ctor.
      * @param first First input
      * @param rest The rest
```

Comment:
@ilyakharlamov What is "the rest"? Let's change it to a better description.

Reply:
@fabriciofx That's not my code. That's from the previous commit. Am I supposed to change the code that was done by somebody else?

Reply:
@ilyakharlamov If the code is wrong, yes.

Reply:
@fabriciofx It wasn't wrong when the code was accepted. Now it suddenly became wrong. Did the rules change since then or what? I fixed that, but I don't think it's fair.

Reply:
@ilyakharlamov It's normal something pass unnoticed. Don't be mad with this. My role as reviewer is look *everything* wrong, did by you or didn't. If you think is out of scope or is a lot of things to do, you can open a PDD.

Reply:
@ilyakharlamov Yeah, there are a lot of `MatcherAssert.assertThat` usage instead of `Assertion.().affirm` but if will write a new test, we recommend use `Assertion.().affirm`.